src/app/WriteClient.h

```diff
@@ -234,10 +340,24 @@ class WriteClient : public Messaging::ExchangeDelegate
     // and mPendingWriteData is populated.
     CHIP_ERROR SendWriteRequest();
 
+    CHIP_ERROR PrepareAttributeIB(const ConcreteDataAttributePath & attributePathParams);
+    CHIP_ERROR FinishAttributeIB();
+    TLV::TLVWriter * GetAttributeDataIBTLVWriter();
+
+    /**
+     * Create a new message (or a new chunk) for the write request.
+     */
+    CHIP_ERROR CreateNewMessage();
```

Comment:
```suggestion
    CHIP_ERROR StartNewMessage();
```

Just to be symmetric with Finalize.